local-tests/setup/tinny-config.ts

```diff
@@ -23,9 +19,9 @@ export const RPC_MAP = {
   [LIT_TESTNET.LOCALCHAIN]: LIT_RPC.LOCAL_ANVIL,
   [LIT_TESTNET.MANZANO]: LIT_RPC.CHRONICLE,
   [LIT_TESTNET.CAYENNE]: LIT_RPC.CHRONICLE,
-  [LIT_TESTNET.DATIL_DEV]: LIT_RPC.YELLOWSTONE,
-  [LIT_TESTNET.DATIL_TEST]: LIT_RPC.YELLOWSTONE,
-  [LIT_TESTNET.DATIL_PROD]: LIT_RPC.YELLOWSTONE,
+  [LIT_TESTNET.DATIL_DEV]: LIT_RPC.CHRONICLE_YELLOWSTONE,
```

Comment:
Why are we prefixing this with `CHRONICLE`

Reply:
It's just the full name of the chain, I think is clearer.

Reply:
`LIT_RPC` from constants already had it this way [here](https://github.com/LIT-Protocol/js-sdk/pull/579/files#diff-f26476a636a35901110ac360f37c1f63adb607f40eb85c6c2f27cc54da0f99dfL669) and that wasn't changed

iirc team discussed and defined to keep the chronicle prefix, don't remember why. This ideally should be consistent with nodes and everything

Reply:
Technically `CHRONICLE` has no association with our `YELLOWSTONE` but we can keep this if it helps with context.